plugins/org.locationtech.udig.tool.edit/src/org/locationtech/udig/tool/edit/internal/messages.properties

```diff
@@ -116,7 +116,7 @@ EditToolPreferences_snapRadius = Snap Radius
 
 EditToolPreferences_vertexDiameter = Vertex Diameter
 
-EditToolPrefernece_fillTransparency = Reduce Fill Transparency (0-100)
+EditToolPrefernece_fillTransparency = Fill Opacity
```

Comment:
Would you refactor this property to `EditToolPreferences_fillOpacity` and update screenshot for EditTool Preferences page (doc).

Reply:
hi @fgdrf,
yes i can do that. I try to fix it this week.

Reply:
the last commit also fixes the misspelling variable for the vertex opacity, so i think #328 is not needed anymore